backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/outputGenerators/DocxUtils.java

```diff
@@ -195,15 +230,22 @@ private static void setText(XWPFRun run, String text) {
         }
     }
 
-    private static void addFormattedText(XWPFRun run, String text, String fontFamily, int fontSize, boolean bold) {
+    private static void addFormattedText(XWPFRun run, String text, String fontFamily, int fontSize, boolean bold, String rrggbbColor) {
         run.setFontSize(fontSize);
         run.setFontFamily(fontFamily);
         run.setBold(bold);
+        if (rrggbbColor != null) {
+            run.setColor(rrggbbColor);
+        }
         setText(run, text);
     }
 
+    private static void addFormattedText(XWPFRun run, String text, int fontSize, boolean bold, String rrggbbColor) {
+        addFormattedText(run, text, "Calibri", fontSize, bold, rrggbbColor);
+    }
+
     private static void addFormattedText(XWPFRun run, String text, int fontSize, boolean bold) {
-        addFormattedText(run, text, "Calibri", fontSize, bold);
+        addFormattedText(run, text, "Calibri", fontSize, bold, null);
```

Comment:
I do not like to have `null` used as passed parameter, to emulate the behaviour of `Optional`

Reply:
I would respectfully disagree here. `Optional` is not designed to be used as method parameter, but rather as a return type. I know it's different in languages that don't have a `null` type, but I don't see any problem using `null` as an acceptable method argument here.